draft-ietf-core-cf-reg-update.md

```diff
@@ -177,10 +179,12 @@ The CoAP Content-Formats registration procedures defined in {{Section 12.3 of -c
 
 | Range | Registration Procedures | Notes |
 |--------|--------|--------|
-| 0-255 | Expert Review | Review procedure described in {{&SELF}}, {{checks}} |
+| 0-255 | Expert Review | Review procedure described in {{&SELF}}, {{checks}}. |
 | 256-9999 | IETF Review with Expert Review or IESG Approval with Expert Review | Review procedure described in {{&SELF}}, {{checks}} |
-| 10000-64999 | Expert Review or First Come First Served (FCFS) | Review procedure described in {{&SELF}}, {{checks}}. <br><br>FCFS is allowed if the registration: <br> * has no parameters, and <br> * has an empty Content Coding, and <br> * the Media Type is not yet used in this registry, and <br> * the Media Type is registered (or approved for registration) in the "Media Types" registry {{IANA.media-types}}. <br><br>Expert Review is required in all other cases. |
-| 65000-65535 | Experimental Use | No operational use
+| 10000-19999 and 33000-64998 | Expert Review | Review procedure described in {{&SELF}}, {{checks}} |
```

Comment:
```suggestion
| 10000-19999 and 33000-64998 | Expert Review | Review procedure described in {{&SELF}}, {{checks}}. |
```

Reply:
Actually I would prefer if we split these dual ranges in two parts, such that the list of "Range" values goes up monotonically.  That would seem more clear to me.

Reply:
I had started like that and then decided that grouping by policy was a better option.  I don’t hold strong opinions, though, so I’m happy to revert.

Reply:
Done in [4458dd3](https://github.com/core-wg/cf-reg-update/pull/80/commits/4458dd31fb3abe7889badb969bd593b51bf08531)